apps/web/pages/signup.tsx

```diff
@@ -161,11 +166,19 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
   const ssr = await ssrInit(ctx);
   const token = z.string().optional().parse(ctx.query.token);
 
+  const isEmailVerifyEnabled = await prisma.feature.findFirst({
+    where: {
+      slug: "email-verification",
+      enabled: true,
+    },
+  });
```

Comment:
Putting this here was a nicer approach than waiting for it to load on the client - it didnt always seem to load in time and disabling the button was weird if you didnt actually have it enabled 

@zomars

Reply:
![Ill Allow It Spanish GIF](https://media4.giphy.com/media/NCjISbEPFxm48/200.gif)

Reply:
I just realized that we can achieve the same effect by just doing:

```
await ssr.viewer.features.map.prefetch();
```

Will test it out and update if it works